packages/fxa-auth-server/lib/senders/emails/templates/cadReminder.ts

```diff
@@ -28,10 +28,11 @@ export const render = () => {
         ${appBadges}
         <mj-section>
           <mj-column>
-            <mj-text css-class="secondary-text">This is an automated email; if you received it in error, no action is required. For more information, please visit Mozilla Support.</mj-text>
+            <mj-text css-class="secondary-text" data-l10n-id="hello-world">This is an automated email; if you received it in error, no action is required. For more information, please visit Mozilla Support.</mj-text>
             <mj-text css-class="secondary-text"><span>Mozilla. 2 Harrison St, #175, San Francisco, CA 94105</span></mj-text>
             <mj-text css-class="secondary-text"><span>Mozilla Privacy Policy</span></mj-text>
             <mj-text css-class="secondary-text"><span>Firefox Cloud Terms of Service</span></mj-text>
+            <mj-text><h3 data-l10n-id="hello-world">Is this localized?</h1></mj-text>
```

Comment:
[fluent/dom](https://github.com/projectfluent/fluent.js/tree/master/fluent-dom) localizes the html elements that have `data-l10n-id` as an attribute. Here, it is able to localize the `h3` tag but nothing happens in case of `mj-text`. And while we can register [custom elements](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements) by extending the HTMLElement class, all mj-elements ultimately get converted over to html.